recipe/meta.yaml

```diff
@@ -1,24 +1,27 @@
+{% set version = "3.1.0" %}
+
 package:
     name: qutip
-    version: 3.1.0
+    version: {{ version }}
 
 source:
-    fn: qutip-3.1.0.tar.gz
-    url: http://qutip.org/downloads/3.1.0/qutip-3.1.0.tar.gz
+    fn: qutip-{{ version }}.tar.gz
+    url: http://qutip.org/downloads/{{ version }}/qutip-{{ version }}.tar.gz
+    sha256: 078399c9830c011ec189db222fcd45f71d2a135800874784d8386d67b4357402
     patches:
       - qutip__init__.patch
-    
+
 build:
     number: 0
     script: $PYTHON setup.py install
```

Comment:
Or maybe not. `bld.bat` is more than that. I guess we need to remove the `script` field and add a `build.sh`.

`bld.bat`:
```bat
"%PYTHON%" setup.py build --compiler=mingw32
if errorlevel 1 exit 1
"%PYTHON%" setup.py build_ext --compiler=mingw32
if errorlevel 1 exit 1
"%PYTHON%" setup.py install
if errorlevel 1 exit 1
```